src/main/java/edu/ie3/datamodel/models/result/system/SystemParticipantResult.java

```diff
@@ -15,12 +15,10 @@
 /** Abstract class that holds values common to all other result entities */
 public abstract class SystemParticipantResult extends ResultEntity {
 
-  /**
-   * @param p active power output normally provided in MW
-   * @param q reactive power output normally provided in MVAr
-   */
+  /** active power output normally provided in MW */
```

Comment:
I think, it's okay, but it might be an idea to link `StandardUnits#ACTIVE_POWER_OUTPUT` (or however this static field is called)?!

Reply:
I just did this because the compiler was complaining about the @param in JavaDoc of a field. I just fixed the JavaDoc to make it work. Not quite sure what addition you have in mind here.